config/environments/production.rb

```diff
@@ -76,4 +76,16 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.action_mailer.smtp_settings =  {
+      address: "smtp.mandrillapp.com",
+      port: 587,
+      domain: "example.com",
+      authentication: "plain",
+      user_name: ENV["MANDRIL_USERNAME"],
+      password: ENV["MANDRIL_PASSWORD"],
+      enable_starttls_auto: true
+  }
+
+  config.action_mailer.default_url_options = { host: ENV['DEFAULT_URL_OPTION'] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.